packages/typescriptlang-org/src/copy/pt.ts

```diff
@@ -0,0 +1,9 @@
+import { defineMessages } from "react-intl"
+import { navCopy } from "./pt/nav"
+import { docCopy } from "./en/documentation"
+import { Copy } from "./en"
+
+export const lang: Copy = defineMessages({
+  ...navCopy,
+  ...docCopy,
+})
```

Comment:
This is how we do language translation in the React side, you have an object which returns keys and values and you incrementally add new references. In this case only the `navCopy` comes from the `pt` subfolder instead of English.

```ts
import { navCopy } from "./pt/nav"
```